src/server/db_slice.cc

```diff
@@ -948,8 +967,9 @@ void DbSlice::PreUpdate(DbIndex db_ind, PrimeIterator it) {
 void DbSlice::PostUpdate(DbIndex db_ind, PrimeIterator it, std::string_view key, bool existing) {
   DbTableStats* stats = MutableStats(db_ind);
 
-  size_t value_heap_size = it->second.MallocUsed();
+  int64_t value_heap_size = it->second.MallocUsed();
   stats->obj_memory_usage += value_heap_size;
+  AddTypeMemoryUsage(it->second.ObjType(), value_heap_size, stats);
   if (it->second.ObjType() == OBJ_STRING)
     stats->strval_memory_usage += value_heap_size;
```

Comment:
I think that strval_memory_usage is redundant now

Reply:
(did that)

Reply:
@romange what do you say about removing strval_bytes from INFO MEMORY ? I guess you added it for debugging tiering. In this PR we expose all the types memory bytes

Reply:
sure thing.

Reply:
removed